### ISSUE TITLE
Calibre doesn't build anymore

### ISSUE THREAD

**Issue by DamienCassou** [NixOS/nixpkgs#97025]
**Describe the bug**

Calibre requires on Python2 and dnspython. nixpkgs recently updated dnspython to version 2 (a7b5b18ab92f4f4b303d3d3526b91a28f627c68c) which requires Python3 (fe23db1fdd2d1edfdf009404bac01cd7d3e84706).

This is similar to https://github.com/NixOS/nixpkgs/pull/96625.

**To Reproduce**

```
nix-build -A calibre
```

**Notify maintainers**

/cc @domenkozar @pSub @AndersonTorres 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.0-10-amd64, Debian GNU/Linux, 10 (buster)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.7`
 - channels(root): `"nixpkgs-20.09pre240426.f9567594d5a"`
 - channels(cassou): `"home-manager, nixpkgs-20.09pre240426.f9567594d5a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by vcunat** [comment]
I'd hope that calibre can work without any python 2 (with 3).

**Comment by FRidh** [comment]
Nope, not yet. Author is not very keen on porting to Python 3, though it does mention Python 3 for Calibre 5, which is not yet released, and will break certain plugins.

We should add a Python 2 version for the required packages, should not be much work.

**Comment by FRidh** [comment]
Fixed with 30f07d16574bd572fa2db4505139e33d2cf36cfe.

**Comment by DamienCassou** [comment]
Thank you!
